src/components/Button/index.tsx

```diff
@@ -125,7 +125,7 @@ export const Button = ({
   let padding;
   switch (size) {
     case "nav":
-      textSize = "text-base font-medium";
+      textSize = "text-base font-normal";
```

Comment:
this needs to be `font-medium` for story.xyz designs.  if you use `size="small"` for OUSD / website I think that would work?